sdk/tests/wallet/backup_restore.rs

```diff
@@ -527,3 +527,44 @@ async fn backup_and_restore_bech32_hrp_mismatch() -> Result<()> {
     );
     tear_down(storage_path)
 }
+
+#[tokio::test]
+// Restore a Stronghold snapshot without secret manager data
```

Comment:
nit but we always but the comment above ;)

Reply:
Not always as you can see in the commit :trollface: But changed